spec/services/next_course_creation_step_service_spec.rb

```diff
@@ -34,9 +36,31 @@
 
       include_examples "next step"
     end
+
+    context "Current step: Accredited body" do
+      let(:current_step) { :accredited_body }
+      let(:expected_next_step) { :entry_requirements }
+
+      include_examples "next step"
```

Comment:
Shared examples can take examples. Can we use that here? That would add a bit of security around calling them.

```suggestion
      include_examples "next step", :accredited_body, :entry_requirements
```